src/components/GraduationCalculatorPage/components/CreditChart/SemesterLectureListModal/index.tsx

```
@@ -1,15 +1,16 @@
 import { startTransition, useState } from 'react';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { authQueries } from 'api/auth/queries';
 import { LectureInfo } from 'api/graduationCalculator/entity';
+import { graduationCalculatorQueries } from 'api/graduationCalculator/queries';
```

ESLint

[error] `api/graduationCalculator/queries` import should occur before import of `api/auth/queries`

(import/order)